INSTALL.md

```diff
@@ -124,12 +124,22 @@ sudo pacman -S base-devel python python-pip \
 
 #### CentOS/Fedora
 
-Installing prerequisites on CentOS 7 or Fedora 25:
+Installing prerequisites on CentOS 8 or Fedora>26:
+
+```
+sudo dnf config-manager --set-enabled PowerTools  # lcms2-devel is in PowerTools repository
+sudo dnf install libtiff-devel libjpeg-devel libzip-devel freetype-devel \
+                 lcms2-devel libwebp-devel tcl-devel tk-devel redhat-rpm-config \
```

Comment:
I can't help but wonder what things like `lcms2-devel`, `tcl-devel`, etc are needed for. We don't seem to have similar requirements for debian.

I realise you've just copied this from the centos 7 instructions, but could I impose on you to confirm if these are really needed?

Reply:
sure, is it enough if I install via pip and start the synapse? Or do I need to do other tests?

Reply:
yeah I think if you can install via pip that's good enough.

Reply:
bare minimum to install is: `dnf install python3` then I have created virtual environment activated the pip and upgrade services.... afterwards installing synapse in Centos8 works fine.

I was able to generate config, activate virtual environments and run synapse without exception been displayed.

I guess the `cms2-devel`, `tcl-devel` could be removed.